config/locales/error.en.yml

```diff
@@ -0,0 +1,19 @@
+en:
+  ea:
+    validation:
+      errors:
+        companies_house_number:
+          hint: An 8 digit number, or 2 letters followed by a 6 digit number
+          invalid_html: Enter a valid number - it should have 8 digits, or 2 letters followed by 6 digits
+              </br>If your number has only 7 digits, enter it with a zero at the start.
```

Comment:
i though by using locales another service can easily over ride the message in their app in their own locale files